ansible-st2/roles/arteria_node/tasks/arteria_runfolder.yml

```diff
@@ -0,0 +1,82 @@
+---
+
+#TODO Ensure all this is versioned!
+
+- name: get arteria-core from github for runfolder
```

Comment:
Maybe smoother to get set it up as a dependency from github in the requirements file instead?